### PR TITLE
doc: Add quick start user guide and development guide

### LINKED ISSUE #283: Add a quick start guide

### PR DESCRIPTION
**Pull request by prashanthpai**
Closes #283 

Signed-off-by: Prashanth Pai <ppai@redhat.com>

### CODE REVIEW

File: doc/quick-start-user-guide.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+```sh
+#  mount -t glusterfs 192.168.56.101:testvol /mnt
+```
+
+> NOTE: IP of any of the two nodes can be used by ReST clients and mount clients.
+
+### Known issues
+
+* Restarting glusterd2 does not restore the cluster
```
Comment (kshlm) [inline doc/quick-start-user-guide.md]: This is fixed with #350 isn't it?

Comment (prashanthpai) [reply doc/quick-start-user-guide.md]: This refers to issue #352

File: doc/quick-start-user-guide.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+$ wget https://github.com/gluster/glusterd2/releases/download/v4.0dev-7/glusterd2-v4.0dev-7-linux-amd64.tar.xz
+$ tar -xf glusterd2-v4.0dev-7-linux-amd64.tar.xz
+```
+
+### Running glusterd2
+
+**Create a working directory:** This is where glusterd2 will store all data which includes logs, pid files, etcd information etc. For this example, we will be using a temporary path. If a working directory is not specified, it defaults to current directory.
+
+```sh
+$ mkdir -p /tmp/gd2-workdir
```
Comment (aravindavk) [inline doc/quick-start-user-guide.md]: `/tmp` is not good idea for working dir example

**Comment by kshlm** [comment]
I'll merge an fix the comments in another PR

**Pull request merged**